cmd/kured/main.go

```diff
@@ -531,7 +533,7 @@ func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []s
 
 		var blockCheckers []RebootBlocker
 		if prometheusURL != "" {
-			blockCheckers = append(blockCheckers, PrometheusBlockingChecker{promURL: prometheusURL, filter: alertFilter})
+			blockCheckers = append(blockCheckers, PrometheusBlockingChecker{filter: alertFilter, includeLabel: alertIncludeLabel})
```

Comment:
I don't think we want to entirely replace the existing prometheus regex blocking functionality, do we?

I would advocate that we preserve the existing functionality if the `--alert-filter-regexp` configuration is passed in, and make this new `--alert-include-label` configuration a mutually exclusive option.

So:

1) add CLI validation that only one configuration is possible (i.e., you aren't allowed to include `--alert-include-label` and `--alert-filter-regexp` at the same time
2) append the `blockCheckers` array above depending on which of the above prometheus configurations are being used

Reply:
Hi @jackfrancis, thanks for your feedback. The first bulletpoint in the PR states that existing code should work... And it is working, or am I missing something perhaps? 
Why shouldn't we be allowed to include both filtering by regex and labels? 

Just to clarify, you want to have the prometheus-client configuration into the PrometheusBlockingChecker struct?

Reply:
Oh, I was confused by the removal of the `promURL` property. I see you discussed that with @evrardjp already. So ignore this comment. :)